src/runtime/rxwin.ri

```diff
@@ -1591,17 +1591,17 @@ int wmap(wbp w)
    /*
     * Get PBuffer for offscreen rendering (OpenGL rendering only)
     */
-   {
-   int pbufattrs[] = {GLX_PBUFFER_WIDTH, DisplayWidth(stddpy, wd->screen),
-                      GLX_PBUFFER_HEIGHT, DisplayHeight(stddpy, wd->screen),
-                      GLX_PRESERVED_CONTENTS, True,
-                      None};
-   ws->pbuf = glXCreatePbuffer(stddpy, wd->configs[0],pbufattrs);
-   if (ws->pbuf == (GLXPixmap)NULL) {
-      /* consider freeing window binding/resources */
-      set_errortext(144);
-      return Failed;
-      }
+   if (wc->rendermode == UGL2D) {
```

Comment:
Suggest adding a comment along the lines of 
> ToDo: Consider moving from Pixel Buffers to Frame Buffer Objects (FBOs)
>       as suggested by https://github.com/XQuartz/XQuartz/issues/264
>       which discusses the problem.
> 
If you want to change/improve the words (for example to mention explicitly that Apple have deprecated Pixel Buffers) then feel free.

Reply:
Thanks @Don-Ward . That is the  issue I looked at when I did my fix. I will add a comment.